### ISSUE TITLE
Raw settings aren't restored to defaults when removed

### ISSUE THREAD

**Issue by mikhaelkh** [uBlockOrigin/uMatrix-issues#113]
### Prerequisites

- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
    - [x] I also searched the existing issues at <https://github.com/gorhill/uMatrix/issues>
- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uMatrix](https://old.reddit.com/r/uMatrix/)
- I tried to reproduce the issue when...
    - [x] uMatrix is the only extension
    - [x] uMatrix with default lists/settings
    - [ ] using a new, unmodified browser profile
- [x] I am running the latest version of uMatrix
- [x] I checked the [documentation](https://github.com/gorhill/uMatrix/wiki) to understand that the issue I report is not a normal behavior
- [x] I used the logger to rule out that the issue is caused by my ruleset

### Description

Raw settings are not restored to its original built-in values when removing them.

### Steps to Reproduce

1. Open `chrome-extension://eckgcipdkhcfghnmincccnhpdmnbefki/dashboard.html#raw-settings`
2. Change `suspendTabsUntilReady` and `disableCSPReportInjection` to `true`, click `Apply changes`
3. Remove lines `suspendTabsUntilReady` and `disableCSPReportInjection`, click `Apply changes`

After that, `suspendTabsUntilReady` and `disableCSPReportInjection` are `true`, but their default values are `false`

### Your environment

* uMatrix version: 1.3.17.1
* Browser Name and version: Chromium 72.0.3626.109 (64-bit) 
* Operating System and version: Windows 10 v1607 x64

**Comment by uBlock-user** [comment]
> After that, `suspendTabsUntilReady` and `disableCSPReportInjection` are `true`, but their default values are `false`

because removal of rules is not possible and previous values are returned. This is by-design.

**Comment by mikhaelkh** [comment]
From https://github.com/gorhill/uMatrix/wiki/Raw-settings
> To restore any one setting to its original built-in value, just remove the whole setting.

**Comment by uBlock-user** [comment]
Yeah and that never worked, if you try to delete any rules, they will return as soon as you click Apply changes, so thought this is by-design.

**Comment by uBlock-user** [comment]
Does it happen on Firefox ?

**Comment by mikhaelkh** [comment]
In uMatrix it worked before. But now apparently the logic is the same as [in uBO](https://github.com/gorhill/uBlock/wiki/Advanced-settings):
> If you want to reset a specific setting to its **defautl** value, just delete the value, uBO will fill the missing value with the default one.
If you want to reset all settings to their default values, delete everything then press "Apply changes".

But deleting everything doesn't reset settings in uMatrix. Only deleting the values does. uBO resets settings when remove the whole line with setting. Can this behavior be unified?
